src/hooks/useMMKV.ts

```diff
@@ -24,14 +24,34 @@ import { getDataType, getInitialValue } from './functions';
  * @param storage The storage instance
  * @returns `useMMKVStorage` hook
  */
-export const create =
-  <T>(storage: MMKVInstance) =>
-  (key: string, defaultValue?: any) => {
+export const create: CreateType =
+  (storage: MMKVInstance) =>
+  <T = undefined>(key: string, defaultValue?: T) => {
     if (!key || typeof key !== 'string' || !storage)
       throw new Error('Key and Storage are required parameters.');
-    return useMMKVStorage<T>(key, storage, defaultValue);
+    if (defaultValue === undefined) {
+      return useMMKVStorage<T>(key, storage);
+    } else {
+      return useMMKVStorage<T>(key, storage, defaultValue);
+    }
```

Comment:
This seems to be redundant. Because if we do not pass the `defaultValue` parameter, it will become undefined still.